Makefile

```diff
@@ -85,3 +87,8 @@ unit: ## Run unit tests (with coverage)
 .PHONY: check-licenses
 check-licenses:
 	$(TEMPDIR)/bouncer check
+
+.PHONY: check-schema
+check-schema: ## Ensure that there aren't any accidental changes to the DB schema version
```

Comment:
❤️

Reply:
You added this :D

Reply:
wow I don't remember doing this at all :self-five?: